bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/overlay/FindReplaceOverlay.java

```diff
@@ -410,7 +416,7 @@ private void applyOverlayColors(Color color, boolean tryToColorReplaceBar) {
 	private void unbindListeners() {
 		getShell().removeShellListener(overlayDeactivationListener);
 		if (targetPart != null && targetPart instanceof StatusTextEditor textEditor) {
-			Control targetWidget = textEditor.getAdapter(ITextViewer.class).getTextWidget();
+			Control targetWidget = Adapters.adapt(textEditor, ITextViewer.class).getTextWidget();
```

Comment:
Are the `textEditor.getAdapter(...`->`Adapters.adapt(textEditor, ...)` changes intended or by accident?

Reply:
This is an accident, very likely stemming from a manual merge

Reply:
Changed in https://github.com/eclipse-platform/eclipse.platform.ui/commit/c7042b7e8f9f3aec27f4d038fa10befb832d8214